### PR TITLE
Release script

### PR DESCRIPTION
**Pull request by scottgonzalez**

### CODE REVIEW

File: build/release.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+module.exports = function( Release ) {
+
+Release.define({
```
Comment (arschmitz) [inline build/release.js]: Spacing

File: build/release.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+	},
+
+	generateArtifacts: function( callback ) {
+		Release.exec( "grunt" );
+		callback( [
+			"dist/PEP.js",
+			"dist/PEP.min.js"
+		] );
+	}
+});
```
Comment (arschmitz) [inline build/release.js]: spacing

File: build/release.js (hunk 1 above)
Comment (scottgonzalez) [inline build/release.js]: Note that this repo doesn't follow the jQuery style guide.

Comment (arschmitz) [inline build/release.js]: Well you have spaced below on lines 14 and 17 so either those should change or these should for consistency. Also the contributing.md https://github.com/jquery/PEP/blob/master/CONTRIBUTING.md specifically says we do follow the jQuery style guide and links to it. So if we are not following  the style guides on contribute.jquery.org we should stop saying we are and linking to them and we should either remove the link to http://contributing.jquery.org and just link to the specific parts we are following ( just CLA? )  or at the very least be explicit in contributing.md which parts we are following.

**Comment by arschmitz** [comment]
:+1: LGTM

**Pull request closed without merging**